src/map/src/items/inventory.cpp

```diff
@@ -52,6 +52,10 @@ inline bool is_spot_correct(const EntitySystem& entitySystem, RoseCommon::Entity
     }
 }
 
+inline bool is_two_handed(uint16_t subtype) {
+    return ((subtype >= ItemSubType::TWOH_SWORD && subtype <= ItemSubType::DUAL_WEILD) && (subtype != ItemSubType::WAND));
```

Comment:
Can we change `DUAL_WEILD` to `DUAL_WIELD` here? (And anywhere else where it's relevant)